services/web/server/src/simcore_service_webserver/products/plugin.py

```diff
@@ -35,24 +24,31 @@
     logger=_logger,
 )
 def setup_products(app: web.Application):
+    #
+    # NOTE: internal import speeds up booting app
+    # specially if this plugin is not set up to be loaded
+    #
+    from ..constants import APP_SETTINGS_KEY
```

Comment:
how much?
for the constants I would expect it to be already everywhere?

Reply:
In reality this will have an effect when i do the import in all setup functions, not just thisone. 
I did not dare doing it all in this PR because of the mocks.

You notice this delay e.g. when you run the CLI. Try this:
- open portainer
- select a container and docker exec inside
- type
```
simcore-service settings
```
- should be instantaneous but it takes a few seconds